opentelemetry/chart/ci/test-values.yaml

```diff
@@ -29,17 +29,26 @@ opentelemetry-operator:
       enabled: false
   kubeRBACProxy:
     enabled: false
-openTelemetry:
+
+opentelemetry:
   openSearchLogs:
     endpoint: test
     username: test
     password: test
   cluster: test
   region: test
+  logsCollector:
+    enabled: true"
+  metricsCollector:
+    enabled: false
   prometheus:
     additionalLabels:
       key1: value1
       key2: value2
+
+    rules:
+      create: false
```

Comment:
at least one rule should be disabled here for the sake of testing. - and of course this value needs to be enabled, otherwise the whole alerting logic is not tested.